tox.ini

```diff
@@ -12,15 +12,17 @@ envlist = py27, py36
 #commands=flake8 iotedgedev
 
 [testenv]
-deps = pytest
+deps = 
+    pytest
+    -rrequirements_dev.txt
 commands = pytest -s -v
 #setenv =
 #    PYTHONPATH = {toxinidir}
 
 #commands = 
 #    python setup.py test
 
-passenv = APPDATA ProgramFiles USERPROFILE PROGRAMDATA DOTENV_FILE LOCALAPPDATA DOTNET_CLI_HOME HOME
+passenv = *
```

Comment:
Why is this line necessary?

Reply:
Because for e2e test cases need more variable. Currently more variables are missed in tox env. So I pass all variables to tox env to fix this issue.